app/bolt_listeners.py

```diff
@@ -61,13 +61,15 @@ def start_convo(
         if TRANSLATE_MARKDOWN:
             new_system_text = slack_to_markdown(new_system_text)
 
+        user_id = context.actor_user_id or context.user_id
+        # Strip bot Slack user ID from initial message
+        msg_text = payload["text"].replace(f"<@{context.bot_user_id}>", "")
```

Comment:
How about replacing trailing spaces too? Perhaps, `re.sub()` works better for it.

Reply:
Good idea